README.md

```diff
@@ -87,11 +87,23 @@ $ curl -H "Content-Type: application/json" \
 ```
 
 You can then update the resource with a **PATCH** request:
+
+Note: in the example file, the `externalIdentifier` and `identification.identifier` must be set to the value returned from the POST, and
+the `tacoIdentifier` and `identification.sdrUUID` must be set to the value retrieved in the GET
+
+Without changing the version number:
+
 ```shell
 $ curl -X PATCH -H "Content-Type: application/json" \ -d@examples/update_request.json \
 -H "On-Behalf-Of: lmcrae@stanford.edu" \ http://localhost:8080/v1/resource/fe1f66a9-5285-4b28-8240-0482c8fff6c7
 ```
 
+With changing the version number:
+```shell
+$ curl -X PATCH -H "Content-Type: application/json" \ -d@examples/update_request_new_version.json \
+-H "On-Behalf-Of: lmcrae@stanford.edu" \ http://localhost:8080/v1/resource/fe1f66a9-5285-4b28-8240-0482c8fff6c7
```

Comment:
Why is there a backslash before `http://localhost:8080/v1/resource`?

Reply:
Copypasta... cleaning up.